tracing-error/src/lib.rs

```diff
@@ -65,8 +74,71 @@
 //!     }
 //! }
 //! ```
-//! In the future, this crate may also provide its own `Error` types as well,
-//! for users who do not wish to use other error-handling libraries.
+//!
```

Comment:
These docs are great, but take or leave the nit: can you add examples showing how these backtraces are printed?